### DISCUSSION TITLE
DOTAv2 - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#5071] [Docs]
# DOTAv2 - Ultralytics YOLOv8 Docs

Delve into DOTA v2, an Oriented Bounding Box (OBB) aerial imagery dataset with 1.7 million instances and 11,268 images.

https://docs.ultralytics.com/datasets/obb/dota-v2/

**Comment by dimpolitik** [comment]
Hi!

I have a dataset that I want to annotate it with orientated bounding boxes. LabelImg and CVAT dont seem to 'support export formats for eight points (x1,y1,x2,y2,x3,y3,y4) annotation of an object as DOTA has. Are you aware which tool was applied for annotation extraction of DOTA?
Thank you for your time.

**Reply by dimpolitik** [reply]
Thank you! Not referred, however, which tool was used for annotating DOTAv2.

**Reply by BeytullahYayla** [reply]
Hey, have you learned which tool have used to label image??

**Reply by pderrenger** [reply]
@BeytullahYayla hello! The DOTA dataset creators have not publicly specified the exact annotation tool they used for labeling their images with oriented bounding boxes. However, they provide a development kit that includes code for converting their annotation format to more common formats like COCO. For your own dataset, you might need to look for specialized annotation tools that support oriented bounding boxes or customize existing tools to suit your needs. The DOTA development kit might also offer some guidance on handling such annotations. Good luck with your project! 😊🛰️

**Comment by voiresa** [comment]
Hi, i am using DOTAv1 to do the train with yolov8 obb, and the convert error occurred for each pictures:
    train: WARNING ⚠️ /home/kxg/fyc/datasets/DOTAvv1/images/train/P2804.png: ignoring corrupt image/label:     could not convert string to float: 'imagesource:GoogleEarth'

I am unable to download dotav2 from the link so I have no idea about how a label text should be here,and whether this error asks me to delete "image source:GoogleEarth"?

**Reply by pderrenger** [reply]
@voiresa hello! It seems like you're encountering an issue with the DOTAv1 dataset while training with YOLOv8 OBB. The error message indicates that there's a problem with parsing the label file associated with the image `P2804.png`. Specifically, it's trying to convert a string (`'imagesource:GoogleEarth'`) to a float, which is not possible.

The label files for YOLOv8 OBB should contain numerical values representing the oriented bounding box annotations, not strings like `'imagesource:GoogleEarth'`. You might need to preprocess your label files to ensure they are in the correct format expected by YOLOv8 OBB.

Here's what you can do:
1. Check the label files for any non-numeric lines or metadata like `'imagesource:GoogleEarth'` and remove them.
2. Ensure that the label files only contain the annotation data in the format expected by YOLOv8 OBB, which typically includes the class index, and the bounding box coordinates and orientation.

If you need guidance on the expected format for the label files, please refer to the Ultralytics documentation for the correct annotation format for oriented bounding boxes. Unfortunately, I can't provide direct links here, but you can find the information under the Datasets section in the Ultralytics Docs.

If you continue to face issues, you might want to consider using the DOTAv2 dataset, which is already formatted for compatibility with YOLOv8 OBB. However, since you mentioned you're unable to download DOTAv2, you may need to manually adjust your DOTAv1 dataset labels to match the expected format.

I hope this helps! If you have further questions or need more assistance, feel free to reach out. 😊🛠️
